src/include/session.h

```diff
@@ -76,6 +76,11 @@ struct WT_COMPILER_TYPE_ALIGN(WT_CACHE_LINE_ALIGNMENT) __wt_session_impl {
 	WT_CURSOR_BACKUP *bkp_cursor;	/* Hot backup cursor */
 	WT_COMPACT	 *compact;	/* Compact state */
 
+	/*
+	 * Lookaside file cursor, sweep thread and eviction worker threads only.
+	 */
+	WT_CURSOR	*las_cursor;	/* Lookaside file cursor */
+
 	WT_DATA_HANDLE *meta_dhandle;	/* Metadata file */
```

Comment:
Maybe we should keep a `meta_cursor` around to match what you are doing with `las_cursor`, rather than caching `meta_dhandle`?